lib/plugins/physics.js

```diff
@@ -117,17 +145,24 @@ function inject (bot, { physicsEnabled }) {
     return dYaw
   }
 
-  function updatePosition (dt) {
-    // If you're dead, you're probably on the ground though ...
-    if (!bot.isAlive) bot.entity.onGround = true
+  function isDead () {
+    if (bot.isAlive === true) deadTicks = 0
+    if (bot.isAlive === false && deadTicks <= 20) deadTicks++
+    if (deadTicks >= 20) return true
+    return false
+  }
+
+  function updatePosition (now) {
+    // Only send updates for 20 ticks after death
+    if (isDead()) return
 
     // Increment the yaw in baby steps so that notchian clients (not the server) can keep up.
     const dYaw = deltaYaw(bot.entity.yaw, lastSentYaw)
     const dPitch = bot.entity.pitch - (lastSentPitch || 0)
 
     // Vanilla doesn't clamp yaw, so we don't want to do it either
-    const maxDeltaYaw = dt * physics.yawSpeed
-    const maxDeltaPitch = dt * physics.pitchSpeed
+    const maxDeltaYaw = PHYSICS_TIMESTEP * physics.yawSpeed
```

Comment:
Why change dt for a constant? What if you want to run mineflayers physics faster? Is this still possible with this change?

Reply:
dt was originally a constant.
This does not change behavior, you can simulate it faster if you decrease the values of PHYSICS_INTERVAL_MS and PHYSICS_SETINTERVAL_MS